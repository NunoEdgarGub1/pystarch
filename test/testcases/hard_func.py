def f(iterable):
    return [y for x, y in iterable]
a = f([('a', 1), ('b', 2)])
