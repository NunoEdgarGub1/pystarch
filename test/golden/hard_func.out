a List(Num)
f Function(iterable: Any)
